server/routes/contact.js

```diff
@@ -1,21 +1,33 @@
 const { asyncMiddleware } = require('../utils/middleware')
-
-module.exports = ({ userAdminService }) => router => {
+const { unwrapResult } = require('../utils/functionalHelpers')
+/**
+ * @typedef {import("../../types/licences").RoService} RoService
+ * @typedef {import("../../types/licences").ResponsibleOfficerResult} ResponsibleOfficerResult
+ * @typedef {import("../../types/licences").ResponsibleOfficer} ResponsibleOfficer
+ */
+/**
+ *
+ * @param userAdminService
+ * @param {RoService} roService
+ * @returns {function(*): *}
+ */
+module.exports = (userAdminService, roService) => router => {
   router.get(
-    '/ro/',
+    '/:borkingId',
     asyncMiddleware(async (req, res) => {
-      const roUsers = await userAdminService.getRoUsers()
-      return res.render('contact/roList', { roUsers })
-    })
-  )
+      // because 'bookingId' is treated specially - see standardRouter.js
+      const { borkingId } = req.params
+      const [ro] = unwrapResult(await roService.findResponsibleOfficer(borkingId, res.locals.token))
```

Comment:
`ro` would be undefined if an error is present?

Reply:
yes. Fixed + test.